pkgs/servers/gemini/stargazer/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromSourcehut, rustPlatform, installShellFiles, scdoc, ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "stargazer";
+  version = "1.0.5";
+
+  src = fetchFromSourcehut {
+    owner = "~zethra";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-n88X3RJD7PqOcVRK/bp/gMNLVrbwnJ2iwi2rCpsfp+o=";
+  };
+
+  cargoSha256 = "sha256-Yqh3AQIOahKz2mLeVNm58Yr6vhjU4aQwN62y3Z5/EJc=";
+
+  doCheck = false; # Uses extenal testing framework that requires network
+
+  nativeBuildInputs = [ installShellFiles scdoc ];
+
+  postInstall = ''
+    scdoc < doc/stargazer.scd  > stargazer.1
+    scdoc < doc/stargazer-ini.scd  > stargazer.ini.5
+    installManPage stargazer.1
+    installManPage stargazer.ini.5
+    installShellCompletion completions/stargazer.{bash,zsh,fish}
+  '';
+
+  meta = with lib; {
+    description = "A fast and easy to use Gemini server";
+    homepage = "https://sr.ht/~zethra/stargazer/";
+    license = licenses.agpl3Plus;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ gaykitty ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ gaykitty ];
    platforms = platforms.linux;
```

Reply:
Does it have a changelog page?

Reply:
Does this count? https://git.sr.ht/~zethra/stargazer/refs